pkg/bgp/speaker/pod_cidr.go

```diff
@@ -30,10 +30,10 @@ var (
 	emptyAdverts = []*metallbbgp.Advertisement{}
 )
 
-func (s *Speaker) withDraw() error {
+func (s *MetalLBSpeaker) withDraw() error {
 	log.Infof("chris withDrawal of all BGP routes")
 	var wg sync.WaitGroup // waitgroup here since we don't care about errors
-	for _, session := range s.PeerSessions() {
+	for _, session := range s.speaker.PeerSessions() {
 		go func(sess metallbspr.Session) { // Need an outer closure to capture session.
 			wg.Add(1)
```

Comment:
There are a few go lint warnings here that we should fix in the original PR, FYI.

Reply:
Cool, I'll make note to lint before final merger.